modules/core/portal-bootstrap/src/main/java/com/liferay/portal/bootstrap/ModuleFrameworkImpl.java

```diff
@@ -756,25 +756,11 @@ private Bundle _addBundle(
 		Properties extraProperties = PropsUtil.getProperties(
 			PropsKeys.MODULE_FRAMEWORK_PROPERTIES, true);
 
-		Parameters extraCapabilitiesParameters = OSGiHeader.parseHeader(
```

Comment:
hey @Preston-Crary   This code was added for this ticket

https://issues.liferay.com/browse/LPS-77350

https://github.com/liferay/liferay-portal/search?p=3&q=LPS-77350&type=Commits

Has this functionality been accounted for?

Reply:
I removed dead code, it's not doing anything useful before my change. It looks like that was left behind by 601fa3a.

Reply:
I see.  So then the commit you mentioned appears wrong.

@shuyangzhou  can you look at this? 
https://github.com/shuyangzhou/liferay-portal/commit/601fa3a46f7a36645361ee41571fe7c0ea322be8 

This seems wrong.  It loses all the extra information that was added.  Previously my code combined the original `provideCapability` combined with the extra capability header values and put them back into the `providedCapability` header.

Reply:
@gamerson see b75fff9